CorrelationVector/tests/CorrelationVectorTests.cpp

```diff
@@ -294,8 +294,17 @@ TEST_CASE("Spin_SortValidation")
 
         lastSpinValue = spinValue;
 
+        // MacOS SYSTEM_CLOCK and std::chrono::system_clock have microsecond precision.
+        // Both Windows and Linux have sub-microsecond precision.
+        // This means we need to wait a bit longer (over 65 milliseconds) on MacOS
+        // for the spin operator to to change the value of the cv.
```

Comment:
nit: typo - there are two "to"s. Also could we keep the casing as "cV" for consistency?

Reply:
Will do! Thanks!